WordPress/UITestsFoundation/Globals.swift

```diff
@@ -24,6 +24,17 @@ public func navigateBack() {
     }
 }
 
+// Generic function for cases where ScreenObject times out due to long loading time
+// making the Editor Screen evaluate to 'nil'
+public func closeEditorDiscardChanges() {
+    let app = XCUIApplication()
```

Comment:
Nitpick: I would be great to inject the app value as a default parameter.

```suggestion
public func closeEditorDiscardChanges(app: XCUIApplication = XCUIApplication()) {
```

There is no _real_ advantage here, because we don't have an `XCUIApplication` instance to pass to the `closeEditorDiscardChanges` call, but IMHO it's a good practice for an object like the `XCUIApplication` instance, which can be easily shared through the tests, as opposed to initialized every time, because the app the tests run on is always the same.